src/components/Card/Card.stories.mdx

```diff
@@ -0,0 +1,529 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Preview,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+import { action } from "@storybook/addon-actions";
+import Button from "../Button/Button";
+import { ButtonTypes } from "../Button/ButtonTypes";
+import Card, { CardImage, CardHeading, CardContent, CardActions } from "./Card";
+import { CardImageRatios, CardImageSizes, CardLayouts } from "./CardTypes";
+import Link from "../Link/Link";
+import Heading from "../Heading/Heading";
+import HorizontalRule from "../HorizontalRule/HorizontalRule";
+import { LinkTypes } from "../Link/LinkTypes";
+import { HeadingDisplaySizes } from "../Heading/HeadingDisplaySizes";
+import Icon from "../Icons/Icon";
+import { IconRotationTypes, IconNames } from "../Icons/IconTypes";
+import Image from "../Image/Image";
+import { getCategory } from "../../utils/componentCategories";
+
+<Meta
+  title={getCategory("Card")}
+  component={Card}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Master?node-id=17167%3A58131",
+    },
+  }}
+  argTypes={{
+    blockName: { table: { disable: true } },
+    className: { table: { disable: true } },
+    modifiers: { table: { disable: true } },
+    id: { table: { disable: true } },
+    layout: { table: { disable: false } },
+    imageAspectRatio: { table: { disable: false } },
+    imageFlip: { table: { disable: false } },
+    border: { table: { disable: false } },
+    padding: { table: { disable: false } },
+    backgroundColor: { control: { type: "color" } },
+    foregroundColor: { control: { type: "color" } },
+  }}
+/>
+
+# Card
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `0.24.0`   |
+
+<Description of={Card} />
+
+Cards are contained units of information that can stand independently from other elements for context. While a Card is a self-contained unit, Cards are often grouped together and displayed in some type of grid layout.
+
+<Preview withToolbar>
+  <Story
+    name="Card Props"
+    args={{
+      id: "custom-card",
+    }}
+  >
+    {(args) => (
+      <Card {...args}>
+        <CardImage
+          src="https://placeimg.com/400/300/animals"
+          alt="Alt text"
+          imageAspectRatio={args.imageAspectRatio}
+        />
+        <CardHeading level={2} id="heading1">
+          Optional Header
+        </CardHeading>
+        <CardHeading level={4} id="heading2">
+          Sollicitudin Lorem Tortor Purus Ornare
```

Comment:
Super super nitpick, but it'd be nice to stick with the Animal Crossing theme somehow.

Reply:
I know nothing about `Animal Crossing`.  I'd be more apt to use `The Hobbit` and `LotR` for inspiration.  That said, I'd take `Animal Crossing` suggestions here.